validator/graffiti/parse_graffiti.go

```diff
@@ -27,6 +44,49 @@ func ParseGraffitiFile(f string) (*Graffiti, error) {
 	if err := yaml.Unmarshal(yamlFile, g); err != nil {
 		return nil, err
 	}
+
+	specific := make(map[types.ValidatorIndex]string, len(g.Specific))
+	for i, o := range g.Specific {
+		specific[types.ValidatorIndex(i)] = ParseHexGraffiti(o)
+	}
+
+	g.Specific = specific
+	g.Default = ParseHexGraffiti(g.Default)
+	g.Ordered = parseGraffitiStrings(g.Ordered)
+	g.Random = parseGraffitiStrings(g.Random)
 	g.Hash = hashutil.Hash(yamlFile)
 	return g, nil
 }
+
+// ParseHexGraffiti checks if a graffiti input is being represented in hex and converts it to ASCII if so
+func ParseHexGraffiti(rawGraffiti string) string {
+	splitGraffiti := strings.SplitN(rawGraffiti, ":", 2)
+	if strings.ToLower(splitGraffiti[0]) == hexGraffitiPrefixKey {
+		target := splitGraffiti[1]
+		if target == "" {
+			log.WithFields(logrus.Fields{
```

Comment:
nit: for single field could just use `log.WithField("graffiti", rawGraffiti).Debu...`